test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -407,7 +405,7 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 			}
 		})
 
-		It("Reverts back all profile configuration", func() {
+		AfterAll("Reverts back all profile configuration", func() {
```

Comment:
What do we gain from the change here?  
The current format will not work as it is:
```
 Unknown Decorator
AfterAll("Reverts back all profile configuration", func() {
/go/src/github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go:408
  [AfterAll] node was passed an unknown decorator: '"Reverts back all profile
  configuration"' 
```

Reply:
Fix. An It cannot be used to return original config